cascade.py

```
@@ -1,146 +1,336 @@
+            help="""Force use of one of the following backends for catflap unlocking/locking:
+              - surepy (use surepy module)
+              - ha (use homeassistant REST/Webhook)""",
+    )
+    args = parser.parse_args()
+
+    CAMERA_ID = args.camera_id if args.camera_id else "default"
+    BACKEND = args.backend if args.backend else None
+    if (CAMERA_ID):
+        cam_cfg = config.CAMERA_OVERRIDES.get(CAMERA_ID)
+        camera_host = urlparse(cam_cfg.get('url')).hostname
+        camera_ssh_username = config.CAMERA_SSH_USERNAME if hasattr(config, "CAMERA_SSH_USERNAME" or not config.CAMERA_SSH_USERNAME in (None, "" , 0)) else None
```

Ruff (0.12.2)

Use `"CAMERA_SSH_USERNAME"` instead of `"CAMERA_SSH_USERNAME" or ...`

Replace with `"CAMERA_SSH_USERNAME"`

(SIM222)